docs/project/set-up-dev-env.md

```diff
@@ -255,25 +255,25 @@ example, you'll edit the help for the `attach` subcommand.
     Your location should be different because, at least, your username is
     different.
 
-3. Open the `api/client/attach.go` file.
+3. Open the `api/client/container/attach.go` file.
 
 4. Edit the command's help message.
 
     For example, you can edit this line:
 
-    ```go
-    noStdin := cmd.Bool([]string{"-no-stdin"}, false, "Do not attach STDIN")
-    ```
+	```go
+	flags.BoolVar(&opts.noStdin, "no-stdin", false, "Do not attach STDIN")
+	```
 
     And change it to this:
 
     ```go
-    noStdin := cmd.Bool([]string{"-no-stdin"}, false, "Do not attach STDIN (standard in)")
+	flags.BoolVar(&opts.noStdin, "no-stdin", false, "Do not attach STDIN (standard in)")
     ```
 
-5. Save and close the `api/client/attach.go` file.
+5. Save and close the `api/client/container/attach.go` file.
 
-6. Go to your running development container.
+6. Rebuild your development container with `make shell`
 
 7. Remake the binary and copy it to `usr/bin`
```

Comment:
We now have a handy `make install`, which copies all the needed binaries to `/usr/bin` in the container. Looks like these docs still refer to `binary`, but there's multiple binaries now